packages/@angular/cli/commands/generate.ts

```diff
@@ -1,28 +1,61 @@
-import * as chalk from 'chalk';
-import * as fs from 'fs';
-import * as os from 'os';
-import * as path from 'path';
+import { cyan, yellow } from 'chalk';
```

Comment:
It seems like this should be a task, rather than changing the build command directly, and that the `new` commands should reuse the task and pass in the different name for the schematics from the config.